src/CommonITILObject.php

```diff
@@ -8405,17 +8406,43 @@ protected function updateActors()
             }
 
            // update actors
+            $common_actor_input = [
+                '_do_not_compute_takeintoaccount' => $this->isTakeIntoAccountComputationBlocked($this->input),
+                '_from_object'                    => true,
+            ];
+            if ($disable_notifications) {
+                $common_actor_input['_disablenotif'] = true;
+            }
             foreach ($added as $actor) {
                 $actor_obj = $this->getActorObjectForItem($actor['itemtype']);
-                $actor_obj->add($actor + [
+                $actor_obj->add($common_actor_input + $actor + [
                     $actor_obj::$items_id_1 => $this->fields['id'], // ex 'tickets_id' => 1
                     $actor_obj::$items_id_2 => $actor['items_id'],   // ex 'users_id' => 1
                     'type'                  => $actortype,
                 ]);
+                if (
```

Comment:
a bunch of comments for the block would be welcomed.

Reply:
It was copied/pasted from `prepareInputForUpdate`.